src/GovUk.Education.ExploreEducationStatistics.Data.Api/Services/PermalinkService.cs

```diff
@@ -210,7 +213,7 @@ private async Task WriteCsvHeaderRow(CsvWriter csv, PermalinkCsvMetaViewModel me
             try
             {
                 var text = await _blobStorageService.DownloadBlobText(
-                    containerName: Permalinks,
+                    containerName: BlobContainers.Permalinks,
```

Comment:
Just wondering why we've removed the static import in favor of specifying the container name as `BlobContainers.Permalinks` ?

Reply:
Ah interesting, yeah I do agree it removes the ambiguity of where stuff is getting imported from.